cmd/traffic/cmd/manager/service.go

```diff
@@ -47,11 +48,10 @@ func (wall) Now() time.Time {
 	return time.Now()
 }
 
-func NewManager(ctx context.Context, env Env) *Manager {
+func NewManager(ctx context.Context) *Manager {
 	ret := &Manager{
 		ctx:   ctx,
 		clock: wall{},
-		env:   env,
```

Comment:
I think this is the issue with the smart agent bc this function uses the config to get the System A info: https://github.com/telepresenceio/telepresence/blob/release/v2/cmd/traffic/cmd/manager/service.go#L100-L102